modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -133,13 +133,16 @@ static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 class Mat;
 GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 GAPI_EXPORTS GMatDesc descr_of(const cv::UMat &mat);
+GAPI_EXPORTS std::vector<GMatDesc> descr_of(const std::vector<cv::Mat> &vec);
+GAPI_EXPORTS std::vector<GMatDesc> descr_of(const std::vector<cv::UMat> &vec);
 #endif // !defined(GAPI_STANDALONE)
```

Comment:
@anton-potapov since you're here, I also wonder if this function should return `GMetaArgs` instead of `std::vector<GMatDesc>`. Was that actually supposed for DLDT use case?

Reply:
[InferenceEngine::PreprocEngine::preprocessWithGAPI](https://github.com/opencv/dldt/blob/30594bb3099b7874f372074e6c0cc82748344435/inference-engine/src/inference_engine/ie_preprocess_gapi.cpp#L335) expects  `std::vector<cv::GMetaArg>` (i.e. `GMetaArgs`) from the ad-hoc version